src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <div class="card__container-image">
+        <img
+        src="images/imac.jpeg"
+        alt="imac_image"
+        class="card__image"
+        >
+      </div>
+
+      <div class="card__container-card-name">
+        <h1 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      </div>
+
+      <div class="card__container-product-code">
+        <h3 class="card__product-code">Product code: 195434</h3>
+      </div>
```

Comment:
Do you really need container for each element?
Also, don't forget that h1 can be only one per page.
I this case it's okay, but you will probably have more that 1 card on page in future
```suggestion
      <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
      
      <h3 class="card__product-code">Product code: 195434</h3>
```

Reply:
I used containers where I needed to use margins to position elements. I did it according to BEM instructions.

Reply:
You don't need an additional wrapper for every element. Just change the display property.